.github/workflows/update_api_references.yml

```
@@ -0,0 +1,46 @@
+name: Update API references
+on:
+  pull_request_review:
+    types: [submitted]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  update_api_references:
+    if: github.event.pull_request.base.ref == 'master' && github.event.review.state == 'approved'
+    name: Update API References
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: |
+          8.0.x
+    - name: install_tool_and_generate_api_docs
+      run: |
+        dotnet tool install -g XMLDoc2Markdown
+        dotnet build ./LLama\LLamaSharp.csproj
```

```suggestion
        dotnet build ./LLama/LLamaSharp.csproj
```

I think this should fix the error